bin/compile

```diff
@@ -31,21 +31,6 @@ branch=""
 dir=$(mktemp -t buildpackXXXXX)
 rm -rf $dir
 
-echo "-----> Configure create-react-app build environment"
-# Set env vars for the inner buildpacks in `.buildpacks`
-# * during compile, install build tooling (devDependencies) with npm & Yarn
-# * in runtime, NODE_ENV is not used (this buildpack launches a static web server)
-export NPM_CONFIG_PRODUCTION=false
```

Comment:
We used to export `NPM_CONFIG_PRODUCTION=true` for the Node buildpack and removed in https://github.com/heroku/heroku-buildpack-nodejs/pull/519 which broke a couple of users who were using this ENV var to detect that they were building on Heroku. 

Ex: https://github.com/heroku/heroku-buildpack-nodejs/pull/519#issuecomment-369746645

I don't think there's a way around this, just wanted to call it out

Reply:
It felt *real good* deleting all the NPM/NODE env monkey business. Now this buildpack utilizes/acts like the Node buildpack as much as possible!